src/libData/AccountData/AccountStore.cpp

```diff
@@ -551,9 +551,20 @@ void AccountStore::RevertCommitTemp() {
   ContractStorage2::GetContractStorage().RevertContractStates();
 }
 
-bool AccountStore::MigrateContractStates(bool ignoreCheckerFailure) {
+bool AccountStore::MigrateContractStates(
+    bool ignoreCheckerFailure, const string& contract_address_output_dir,
+    const string& normal_address_output_dir) {
   LOG_MARKER();
 
+  std::ofstream os_1;
+  std::ofstream os_2;
```

Comment:
Can define as osContract and osNormal? It will be easier to read.

Reply:
okay maybe i'll change once the current image was built